.gitignore

```diff
@@ -9,3 +9,16 @@ erl_crash.dump
 
 # We have example files for docker-compose.yml so let's ignore it
 docker-compose.yml
+
+/node_modules
+
+# Ignore in umbrella apps node modules
+apps/*/node_modules/
+
+# Ignore generated files of in umbrella apps
```

Comment:
good let's use that instead of trying to guess from the top all the things we want to ignore from.